code/_core/obj/item/weapon/melee/ling_blade.dm

```diff
@@ -6,7 +6,7 @@
 	icon = 'icons/obj/item/weapons/melee/swords/arm_blade.dmi'
 	damage_type = /damagetype/melee/sword/armblade
 
-	value = -1
+	value = 5000
```

Comment:
The armblade shouldn't be in rotation as it is attached to the ling suit; a boss drop.

Reply:
alright, reverted